packages/vertica-nodejs/lib/type-overrides.js

```diff
@@ -20,6 +20,22 @@ function TypeOverrides(userTypes) {
   this._types = userTypes || types
   this.text = {}
   this.binary = {}
+  // this is a 'temporary' solution allowing us to continue to use pg-types as long as we can to avoid another large 
```

Comment:
We should create a task for creating a v-types package, if it hasn't been created yet.